centrallix/tests/test_cxss-credDB_00.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+#include <stdio.h>
+#include "cxss/credentials_db.h"
+
+long long
+test(char** name)
+    {
+     *name = "CXSS Cred DB 00: Basic Init";
+
+    
+    /** Basic test of init and release */
+    char * PATH = "/home/devel/test.db";
```

Comment:
Be sure to not have an install-dependent path in the tests.  There may not always be a user "devel".  You may want to use a relative path in the current folder instead of an absolute path.